README.md

```diff
@@ -1,16 +1,53 @@
 # Transformer Models for MATLAB
-[![CircleCI](https://img.shields.io/circleci/build/github/matlab-deep-learning/transformer-models?label=tests)](https://app.circleci.com/pipelines/github/matlab-deep-learning/transformer-models)
```

Comment:
I lost this change when updating. I'll add it back (although it'll just say tests failing right now 😢)